chemprop/v2/tests/unit/featurizers/test_cgr.py

```diff
@@ -168,8 +168,8 @@ def test_keys(self, available_rxn_mode_names):
         """
         Test that the keys function returns the correct set of modes.
         """
-        assert RxnMode.keys() == set(
-            available_mode.lower() for available_mode in available_rxn_mode_names
+        assert set(RxnMode.keys()) == set(
+            available_mode.upper() for available_mode in available_rxn_mode_names
```

Comment:
`available_rxn_mode_names` are already capitalized? So you don't need `.upper()`, but I guess it doesn't hurt.

Reply:
doesn't hurt and then isolates this test to checking _only_ the name not the capitalization scheme we use on the backend